### DISCUSSION TITLE
Judol

### DISCUSSION THREAD

**Discussion by lokamaya** [ABPindo/indonesianadblockrules#605] [Ideas]
Apakah ABPindo akan memblock situs judi online? Semoga, ya.

**Comment by gmarcher** [comment]
Ya benar
Secara default, ABPindo melakukan blokir elemen iklan baik itu judi online atau iklan dewasa. Namun untuk hasil maksimal, pengguna tetap disarankan menggunakan DNS yang memblokir judi online, seperti adguard DNS, openDNS family atau DNS yang patuh trust positif (jika tinggal di Indonesia).
